src/WebUI/Resources.js

```diff
@@ -3,7 +3,7 @@
 define("Environments/Providers/Kubernetes/Resources", ["require", "exports"], function (require, exports) {
     "use strict";
     Object.defineProperty(exports, "__esModule", { value: true });
-    exports.AgeText = "Age";
+    exports.AgeText = "Created";
     exports.NodeNameText = "Node Name";

```

Comment:
as we are working on taking care of all of the small issues.
"Node Name" should be "Node name", if we are not using this name, remove this

Reply:
removed